### PR TITLE
Menus fixes for the aspect ratio

### PR DESCRIPTION
**Pull request by TheBigEye**
- Menus and some GUI components now adjust to aspect ratio
- Moved the Skins Entry to the Options Menu to reduce the amount of space taken up by the Title display options
- Fixed Resource packs and Skins menu, now its components are positioned correctly
- Fixed inventory search tip position
- Fixed Book gui height
- Fixed a bug regarding level rendering on screen

### CODE REVIEW

**Review by Litorom — APPROVED** [review]
It’s looks good to me. Waiting for @Makkkkus

File: src/main/java/minicraft/screen/SkinDisplay.java [hunk 1]
```diff
@@ -214,7 +219,7 @@ public void render(Screen screen) {
 
 		// Help text.
 		Font.drawCentered("Use "+ Game.input.getMapping("cursor-down") + " and " + Game.input.getMapping("cursor-up") + " to move.", screen, Screen.h - 17, Color.DARK_GRAY);
-		Font.drawCentered(Game.input.getMapping("SELECT") + " to select, and " + Game.input.getMapping("EXIT") + " to cancel." , screen, Screen.h - 9, Color.DARK_GRAY);
+		Font.drawCentered(Game.input.getMapping("SELECT") + " to select, " + Game.input.getMapping("EXIT") + " to cancel." , screen, Screen.h - 9, Color.DARK_GRAY);
```
Comment (Makkkkus) [inline src/main/java/minicraft/screen/SkinDisplay.java:222]: This is gramatically incorrect.

File: src/main/java/minicraft/screen/TitleDisplay.java [hunk 2]
```diff
@@ -44,7 +44,6 @@ public TitleDisplay() {
 				else Game.setDisplay(new WorldGenDisplay());
 			}),
 			new SelectEntry("Options", () -> Game.setDisplay(new OptionsMainMenuDisplay())),
-            new SelectEntry("minicraft.display.skin", () -> Game.setDisplay(new SkinDisplay())),
```
Comment (Makkkkus) [inline src/main/java/minicraft/screen/TitleDisplay.java:47]: Is this required?

File: src/main/java/minicraft/screen/SkinDisplay.java (hunk 1 above)
Comment (TheBigEye) [reply src/main/java/minicraft/screen/SkinDisplay.java:222]: did not fit the screen

Comment (Makkkkus) [reply src/main/java/minicraft/screen/SkinDisplay.java:222]: Split it into two lines

Comment (Litorom) [reply src/main/java/minicraft/screen/SkinDisplay.java:222]: It’s for space reasons. Otherwise it goes off the screen.

Comment (Litorom) [reply src/main/java/minicraft/screen/SkinDisplay.java:222]: I encountered this too

File: src/main/java/minicraft/screen/TitleDisplay.java (hunk 2 above)
Comment (TheBigEye) [reply src/main/java/minicraft/screen/TitleDisplay.java:47]: I could use conditionals and re-adjust the title screen, it's just that there were too many options and I saw it as a simple solution

Comment (Makkkkus) [reply src/main/java/minicraft/screen/TitleDisplay.java:47]: Okay

**Pull request merged**